find_attitude/find_attitude.py

```diff
@@ -26,15 +23,37 @@
 try:
     AGASC_PAIRS_FILE = os.environ["AGASC_PAIRS_FILE"]
 except KeyError:
-    AGASC_PAIRS_FILE = ska_path(
-        "data", "find_attitude", "distances-kadi-local.h5"
-    ) or ska_path("data", "find_attitude", "distances.h5")
+    if (path := SKA / "data" / "find_attitude" / "distances-kadi-local.h5").exists():
```

Comment:
I'm not clear on what this is - so should it get cut or does it belong someplace in docs?

Reply:
I think that at one point that was a link to a file local on web-kadi or kady. Performance was better with this, but it looks like it doesn't exist now on either machine so we could just drop this special case.
```
(base) web-kadi$ cd ~/ska/data/find_attitude/
(base) web-kadi$ ls
distances.h5  distances-kadi-local.h5@
(base) web-kadi$ ls -l
total 1316880
-rw-rw-r-- 1 aldcroft aspect 1343185765 Mar 16  2015 distances.h5
lrwxrwxrwx 1 aldcroft aspect         43 Feb 16  2015 distances-kadi-local.h5 -> /export/tom/data/find_attitude/distances.h5
(base) web-kadi$ ls /export/tom/data/find_attitude/distances.h5
ls: cannot access '/export/tom/data/find_attitude/distances.h5': No such file or directory
```

Reply:
And if you end up wanting to special case it in the future for speed, it would probably be fine to just handle that by changing the find_attitude data symlink without a code change
```
jeanconn-web-kadi> ls -ld /export/servers/kadi/ska3/data/find_attitude
lrwxrwxrwx 1 kadi kadi 32 Aug 14  2023 /export/servers/kadi/ska3/data/find_attitude -> /proj/sot/ska/data/find_attitude/
```

Reply:
I took this code out.